docs/source/usage/param/particles/current.rst

```diff
@@ -0,0 +1,64 @@
+.. _usage-params-core-currentdeposition:
+
+Current Deposition
+""""""""""""""""""
+
+The current solver can be set in :ref:`species.param <usage-params-core>` or directly per species :ref:`speciesDefinition.param <usage-params-core>`.
+
+The following operations can be applied in the ``picongpu::particles::InitPipeline`` of the latter:
```

Comment:
It should be `speciesDefinition.param`

> Should one add recommendations here which strategy is best suited for specific hardware, even if by example?

For that we need to get first an overview how the strategies perform on different hardware. I only change the default for NVIDIA because I tested P100 and V100.